### DISCUSSION TITLE
Would be useful to have a 3D spring joint

### DISCUSSION THREAD

**Discussion by blackears** [godotengine/godot-proposals#4864] [3D]
At the moment, Godot has a spring joint for 2D but not 3D.  I've seen some discussion that you can get spring behavior using a Generic6DOFJoint, but this joint is not well documented and I can't find any good examples of how to set it up.  Even if it is possible with a Generic6DOFJoint, I would feel that spring behavior is common enough that a dedicated spring joint would be useful all on its own.

**Comment by smoothbrainsoftware** [comment]
yeah this is much needed in the editor

**Reply by Calinou** [reply]
Please don't bump issues without contributing significant new information. Use the :+1: reaction button on the first post instead.
